### PR TITLE
Report RF band as a string

### PR DESCRIPTION
**Pull request by mattludwigs**
This change uses the libqmi source as reference on how to parse the band
integer into a string.

This provides a more user friendly understanding of the active band for
the radio interface.

### CODE REVIEW

File: lib/qmi/codec/network_access.ex [hunk 1]
```diff
@@ -432,8 +432,71 @@ defmodule QMI.Codec.NetworkAccess do
...
+          }
+        ]
 
     parse_radio_band_information(rifs, n - 1, rest)
   end
+
+  # Parsing logic uses libqmi enumeration definition as reference
+  # https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/blob/master/src/libqmi-glib/qmi-enums-nas.h#L173-269
+  defp parse_active_band(band) when band >= 0 and band <= 19 do
+    "CDMA Band Class #{inspect(band)}"
```
Comment (fhunleth) [inline lib/qmi/codec/network_access.ex:449]:
```suggestion
    "CDMA Band Class #{band}"
```

File: lib/qmi/codec/network_access.ex [hunk 2]
```diff
@@ -432,8 +432,71 @@ defmodule QMI.Codec.NetworkAccess do
...
+  defp parse_active_band(band) when band >= 120 and band <= 162 do
+    "LTE Band #{lte_band_name(band)}"
+  end
+
+  defp parse_active_band(band) when band >= 200 and band <= 205 do
+    "TDS-CDMA Band #{tds_cdma_band_name(band)}"
+  end
+
+  defp parse_active_band(band) do
+    "Active Band #{inspect(band)}"
```
Comment (fhunleth) [inline lib/qmi/codec/network_access.ex:469]:
```suggestion
    "Active Band #{band}"
```

File: lib/qmi/codec/network_access.ex (hunk 1 above)
Comment (mattludwigs) [reply lib/qmi/codec/network_access.ex]: @fhunleth updated!

File: lib/qmi/codec/network_access.ex (hunk 2 above)
Comment (mattludwigs) [reply lib/qmi/codec/network_access.ex]: @fhunleth updated!

**Pull request merged**